test/service/test_tdr.py

```diff
@@ -89,8 +89,10 @@ def test(dataset: BigQueryDataset):
             test(BigQueryDataset('test-project', 'name', False))
 
     def test_tdr_dataset_config(self):
-        dataset = config.tdr_bigquery_dataset
-        self.assertNotEqual(dataset.is_snapshot, dataset.name.startswith('datarepo_'))
+        from deployments.sandbox.environment import env as sandbox_env
```

Comment:
I wouldn't use a real snapshot. It shouldn't matter if this is a unit test.

Also, you can't import like this. It's a fluke that the project root is on the python path.